app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```

Comment:
If the root path points to users#show, the params won't contain an id key, so you'll need to use the `current_user` instead.

How about this?

```
def show
  user_id = params[:id] || current_user.id
  @user = User.find(user_id)
end
```